tests/pageobjects/PreJoinScreen.ts

```diff
@@ -0,0 +1,31 @@
+import PreMeetingScreen from './PreMeetingScreen';
+
+const DISPLAY_NAME_ID = 'premeeting-name-input';
+const JOIN_BUTTON_TEST_ID = 'prejoin.joinMeeting';
+
+/**
+ * Page object for the PreJoin screen.
+ */
+export default class PreJoinScreen extends PreMeetingScreen {
+    /**
+     * Returns the join button element.
+     */
+    getJoinButton(): ChainablePromiseElement {
```

Comment:
Is this method used somewhere? I don't seem to find any references.

Reply:
Same.